sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py

```diff
@@ -207,6 +207,49 @@ def callback(raw_response, _, headers):
         # Check page metadata
         self.assertFormPagesTransformCorrect(invoice.pages, read_results)
 
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_invoice_stream_multipage_transform_pdf(self, client):
+        responses = []
+
+        def callback(raw_response, _, headers):
+            analyze_result = client._deserialize(AnalyzeOperationResult, raw_response)
+            extracted_invoice = prepare_prebuilt_models(analyze_result)
+            responses.append(analyze_result)
+            responses.append(extracted_invoice)
+
+        with open(self.multipage_vendor_pdf, "rb") as fd:
+            myfile = fd.read()
+
+        poller = client.begin_recognize_invoices(
+            invoice=myfile,
+            include_field_elements=True,
+            cls=callback
+        )
+
+        result = poller.result()
+        raw_response = responses[0]
+        returned_models = responses[1]
+        read_results = raw_response.analyze_result.read_results
+        document_results = raw_response.analyze_result.document_results
+        page_results = raw_response.analyze_result.page_results
+
+        self.assertEqual(1, len(returned_models))
+        returned_model = returned_models[0]
+        self.assertEqual(2, len(returned_model.pages))
+        self.assertEqual(1, returned_model.page_range.first_page_number)
+        self.assertEqual(2, returned_model.page_range.last_page_number)
+
+
+        self.assertEqual(1, len(document_results))
+        document_result = document_results[0]
+        self.assertEqual(1, document_result.page_range[0])  # checking first page number
+        self.assertEqual(2, document_result.page_range[1])  # checking last page number
+
+        self.assertInvoiceTransformCorrect(returned_model, document_result.fields, read_results)
+
+        self.assertFormPagesTransformCorrect(returned_model.pages, read_results)
```

Comment:
```suggestion
        self.assertFormPagesTransformCorrect(returned_model.pages, read_results, page_results)
```

We should pass in the page_results here so we can check the table transform